test/modules/aws-vpc-routetable-integration.ts

```diff
@@ -135,7 +135,7 @@ describe('RouteTable Integration Testing', () => {
     query(
       `
       INSERT INTO subnet (availability_zone, vpc_id, cidr_block, region)
-      SELECT '${region}a', id, '10.${randIPBlock}.1.0/24', '${region}'
+      SELECT (SELECT name FROM availability_zone WHERE region = '${region}' ORDER BY 1 DESC LIMIT 1), id, '10.${randIPBlock}.1.0/24', '${region}'
```

Comment:
`Order by 1`?

Reply:
It will order by the first column that is being returned. In this case, would be equivalent to:
```suggestion
      SELECT (SELECT name FROM availability_zone WHERE region = '${region}' ORDER BY name DESC LIMIT 1), id, '10.${randIPBlock}.1.0/24', '${region}'
```

Reply:
Wtfsql 😅

Reply:
That pattern is used a lot in SQL injection because we don't know the column names :))